generators/app/templates/src/main/resources/application-kafka.yml.ejs

```diff
@@ -0,0 +1,37 @@
+kafka:
+  bootstrap.servers: localhost:9092
+<%_ if (pollingTimeout) { _%>
+  polling.timeout: <%= pollingTimeout %>
+<%_ } _%>
+<%_ if (components.includes('consumer')) { _%>
+  consumer:
+    <%_ for (entity of entities) { _%>
+    <%= _.camelCase(entity) _%>:
+      # This is a template topic naming convention which can be changed.
+      # %3Cmessage_type%3E.%3Capplication_name%3E.%3Centity_name%3E with (all in snake_case):
+      # - %3Cmessage_type%3E: queuing, logging, tracking, etl/db, streaming, push, user...
+      # - %3Capplication_name%3E: the application base name
+      # - %3Centity_name%3E: the entity name which is consumed
```

Comment:
As we are now using a template I would put this block only once just above `consumer:` at the same level just for information. What do you think?

You could replace `This is a template topic naming convention which can be changed.` by `There a topic naming convention below which can be changed.`